chipy_org/settings.py

```diff
@@ -170,13 +170,10 @@ def env_list(key, defaults=None, delimiter=','):
 LOGIN_ERROR_URL = '/'
 
 AUTHENTICATION_BACKENDS = [
-    'social_auth.backends.twitter.TwitterBackend',
-    'social_auth.backends.facebook.FacebookBackend',
-    'social_auth.backends.google.GoogleOAuth2Backend',
-    'social_auth.backends.browserid.BrowserIDBackend',
-    'social_auth.backends.contrib.linkedin.LinkedinBackend',
```

Comment:
Are we dropping facebook and linkedin?

Reply:
Yeah, we only support openid, google, githhub right now.